### PR TITLE
Add com.github.gabutakut.gabutdm

### PR DESCRIPTION
**Pull request by torikulhabib**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/gabutakut/gabutdm
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.gabutakut.gabutdm.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - '*.la'
+  - '*.o'
+  - '/include'
+modules:
+  - name: libcanberra
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: git://git.0pointer.net/libcanberra.git
+        tag: master
```
Comment (hfiguiere) [inline com.github.gabutakut.gabutdm.yml:25]: `master` isn't a tag. It's a branch. You should have a specific commit. Or better use `shared-modules` as it has all that is needed to run it.

File: com.github.gabutakut.gabutdm.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - '--talk-name=com.canonical.Unity.LauncherEntry'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=x11'
+  - '--filesystem=xdg-download'
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '*.o'
```
Comment (hfiguiere) [inline com.github.gabutakut.gabutdm.yml:17]:
```suggestion
```

File: com.github.gabutakut.gabutdm.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: com.github.gabutakut.gabutdm
+finish-args:
+  - '--share=network'
+  - '--talk-name=com.canonical.Unity.LauncherEntry'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=x11'
```
Comment (hfiguiere) [inline com.github.gabutakut.gabutdm.yml:12]:
```suggestion
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.github.gabutakut.gabutdm

File: com.github.gabutakut.gabutdm.yml (hunk 1 above)
Comment (hfiguiere) [reply com.github.gabutakut.gabutdm.yml:23]: this wasn't resolved.

**Comment by nedrichards** [comment]
bot, build com.github.gabutakut.gabutdm

**Comment by hfiguiere** [comment]
There is still an unadressed comment, albeit marked as resolved

File: com.github.gabutakut.gabutdm.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  - '*.a'
+  - '*.la'
+  - '/include'
+modules:
+  - name: libcanberra
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: git://git.0pointer.net/libcanberra.git
+        tag: shared-modules
```
Comment (hfiguiere) [inline com.github.gabutakut.gabutdm.yml:23]: this is not what I said at all.

Comment (hfiguiere) [reply com.github.gabutakut.gabutdm.yml:23]: See https://github.com/flathub/flathub/wiki/App-Maintenance#required-files for some info

**Comment by barthalion** [comment]
bot, build com.github.gabutakut.gabutdm

**Comment by torikulhabib** [comment]
Libcanberra just working use tag: master🥲

**Comment by hfiguiere** [comment]
> Libcanberra just working use tag: master🥲

I said previously to use the existing shared-module that has all that is needed to build.
`master` is NOT  a tag.

**Comment by torikulhabib** [comment]
bot, build com.github.gabutakut.gabutdm

**Pull request closed without merging**